run.sh

```diff
@@ -34,11 +34,16 @@ package_full="${package_name}-${package_version}"
 package_full_ll="${package_name}_${package_version}"
 echo "Building " $package_name " version " $deb_version
 
+git_clone_args=""
+if [[ ${package_name} == "ti-lvgl-demo" ]] ; then
```

Comment:
Hacky hardcoded package name checks, yuk. Why not have this "clone args" defined inside `ti-lvgl-demo/version.sh`?